[2014-03-18 11:23:59] log.ERROR: exception 'ErrorException' with message 'Undefined variable: incomedescription' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:912 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(912): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 912, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->postEdit('1018') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postEdit', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postEdit', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-18 11:24:26] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/storage/views/34701f2e9519f64aa8936ae800168eb2:490 Stack trace: #0 /var/www/arp_dev/app/storage/views/34701f2e9519f64aa8936ae800168eb2(490): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 490, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getEdit', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getEdit', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-18 12:14:50] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42803]: Grouping error: 7 ERROR:  column "event.event" must appear in the GROUP BY clause or be used in an aggregate function' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:274 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(274): PDOStatement->execute(Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'select count(ev...', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select count(ev...', Array, Object(Closure)) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select count(ev...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(713): Illuminate\Database\Query\Builder->get() #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(86): Controllers\Admin\MainvoiceController->getVE('1002', '2010-01-01', '2014-03-18') #8 [internal function]: Controllers\Admin\MainvoiceController->getShow('1018') #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} Next exception 'Exception' with message 'SQLSTATE[42803]: Grouping error: 7 ERROR:  column "event.event" must appear in the GROUP BY clause or be used in an aggregate function (SQL: select count(event.id) as number, event from "event" inner join "voucher_event" on "event"."id" = "voucher_event"."event_id" where "merchant_id" = ? and "datetimecreated" >= ? and "datetimecreated" <= ? and "event_id" >= ? and "event_id" <= ? group by "event_id") (Bindings: array (   0 => '1002',   1 => '2010-01-01',   2 => '2014-03-18',   3 => 490,   4 => 499, ))' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select count(ev...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select count(ev...', Array, Object(Closure)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select count(ev...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(713): Illuminate\Database\Query\Builder->get() #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(86): Controllers\Admin\MainvoiceController->getVE('1002', '2010-01-01', '2014-03-18') #7 [internal function]: Controllers\Admin\MainvoiceController->getShow('1018') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-18 12:25:55] log.ERROR: exception 'ErrorException' with message 'Undefined variable: numVe' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:88 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(88): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 88, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getShow('1018') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-18 12:31:02] log.ERROR: exception 'ErrorException' with message 'Undefined variable: numve' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:88 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(88): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 88, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getShow('1018') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-18 12:31:10] log.ERROR: exception 'ErrorException' with message 'Undefined variable: eventnum' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:90 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(90): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 90, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->getShow('1018') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-03-18 12:54:48] log.ERROR: exception 'ErrorException' with message 'Undefined property: stdClass::$sumevents' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:239 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(239): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 239, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(85): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getShow('1018') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getShow', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1018') #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
